### DISCUSSION TITLE
Move a record

### DISCUSSION THREAD

**Discussion by alanfallow** [Budibase/budibase#10985] [Help] [answered]
Hi folks 
Is it possible to setup a button etc to move a record from 1 table to another.  Using gsheets or excel 
Thank you

**Comment by melohagan** [comment, marked as answer]
Hey @alanfallow 

Yes this is possible! There's a few ways to go about it. Here's some docs pages that could be useful to read through:

- [Execute query](https://docs.budibase.com/docs/data-actions#execute-query) Allows you to pass data into a query and execute it
- [Action chaining](https://docs.budibase.com/docs/chaining-actions) Allows you to use the response of one query as the input of another
- [Data provider](https://docs.budibase.com/docs/data-provider) Can be filtered to return a record
- [Automations](https://docs.budibase.com/docs/automation-steps) Setup a workflow for moving one record to another - explore the action steps for more info.

**Reply by alanfallow** [reply]
Ok thanks will have a scroll through it.  Is it beneficial to  upgrade to the paid version.
Thanks again.

**Reply by melohagan** [reply]
> Is it beneficial to upgrade to the paid version.

We certainly think so! While you can indeed build fantastic apps with the free-cloud or community edition of Budibase; once it comes to the administration of those apps within a business the paid for features will add lots of value.

With that said "Beneficial" is subjective. Please feel free to contact our sales team (sales@budibase.com) so that the specific benefits for you can be discussed, if that'd be helpful?

The outline of features can be found on the pricing page: https://budibase.com/pricing/
